app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java

```diff
@@ -20,11 +26,28 @@ public ParkingView(Activity activity, ActivityMainBinding binding) {
     public void showDialogFragment(ListenerDialogFragment dialogFragmentListener) {
         ParkingDialogFragment dialog = ParkingDialogFragment.newInstance(dialogFragmentListener);
         FragmentManager fragmentManager = getFragmentManager();
-        dialog.show(fragmentManager, ParkingDialogFragment.TAG);
+        dialog.show(fragmentManager, Constants.PARKING_DIALOG_FRAGMENT);
     }
 
     @Override
     public void showNumberOfParkingAvailable(String numberOfParkingAvailable) {
         binding.textViewMainActivity.setText(numberOfParkingAvailable);
     }
+
+    public void showReservationActivity() {
+        Context context = getContext();
+        Activity activity = getActivity();
+        if (context != null && activity != null) {
```

Comment:
you can use only the activity instance, instead of activity and context

Reply:
Done